tests/integration-tests/conftest.py

```diff
@@ -460,6 +472,19 @@ def setup_sts_credentials(region, request):
     unset_credentials()
 
 
+def get_az_id_to_az_name_map(region, credential):
+    """Return a dict mapping AZ IDs (e.g, 'use1-az2') to AZ names (e.g., 'use1-az1')."""
```

Comment:
minor: the comment is wrong, in the second part we should have the name as example.